apps/login/app/(login)/signup/page.tsx

```diff
@@ -0,0 +1,27 @@
+import {
+  getLegalAndSupportSettings,
+  getPasswordComplexitySettings,
+  server,
+} from "#/lib/zitadel";
+import RegisterForm from "#/ui/RegisterForm";
+
+export default async function Page() {
```

Comment:
Is this page redundant? What's the difference between /register and /signup?

Reply:
one allows the user to set the auth method and one is the default register function which already shows the password

Reply:
I don't get why we need a default register function which already shows the password. If we need a preselection, why not preselect the passkey? For me, the difference between the terms signup and register is obscure.

Reply:
This is just because I kind of wanted to deprecated the old functionallity